src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -82,6 +82,8 @@
     private final Launcher launcher;
     private final @CheckForNull Node node;
     private final @CheckForNull String toolName;
+    private Boolean isDockerInitialized = false;
+    private Boolean isDockerOnWindows = false;
```

Comment:
Please use `boolean` if you do not consider using the `null` value. If you really intend to use `Boolean`, one field is enough here